src/lib/get-topper-settings.js

```diff
@@ -44,12 +46,20 @@ const followPlusDigestEmail = (flags) => {
 	return flags.onboardingMessaging === 'followPlusEmailDigestTooltip';
 };
 
+const useLiveBlogV2 = () => {
+	return {
+		largeHeadline: true,
+		backgroundColour: 'paper',
+		modifiers: ['full-bleed-offset']
+	}
+};
+
 const useLiveBlogOrPackageTopper = (content, flags) => {
 	const designTheme =
 		(content.package && content.package.design.theme) ||
 		(content.design && content.design.theme);
 	const isStandaloneLiveBlog =
-		!content.package && content.realtime && content.liveBlog;
+		content.type === 'article' && content.realtime;
```

Comment:
As we need to do establish if the content item is a v1 live blog in `isLiveBlogOrPackage()`, I'd suggest this expression into an `isLiveBlogV1()` function (much like the `isLiveBlogV2()` function). This `isLiveBlogV1()` function could then be used here and in `isLiveBlogOrPackage()`.